app/services/signer/eth/signer.go

```diff
@@ -21,3 +22,13 @@ func NewEthSigner(privateKey string) *Signer {
 func (s *Signer) Sign(msg []byte) ([]byte, error) {
 	return crypto.Sign(msg, s.privateKey)
 }
+
+func PrivateToPublicKeyToAddress(privateKey string) common.Address {
```

Comment:
Attach func to struct Signer with name `Address() common.Address` and add signer to `ConsensusMessageHandler` struct